ci/jenkins/test-mc.sh

```diff
@@ -295,10 +295,11 @@ function deliver_antrea_multicluster {
     # be leveraged successfully
     chmod -R g-w build/images/ovs
     chmod -R g-w build/images/base
+    image_tar_name="antrea-ubuntu.tar"
```

Comment:
Not necessary to define this variable?

Reply:
Do you mean this code refactor unrelated to this pr?

Reply:
+1, Is this change related to this PR?